### PR TITLE
Kkraune/partial updates more details

### PR DESCRIPTION
**Pull request by kkraune**
@geirst I think I will update the illustration at https://docs.vespa.ai/en/attributes.html with a "D" tensor attribute for completeness in a separate PR, and improve linkage to this new partial update document

@jobergum FYI

### CODE REVIEW

File: en/partial-updates.html [hunk 1]
```diff
@@ -149,9 +163,17 @@ <h2 id="write-pipeline">Write pipeline</h2>
       More details in <a href="attributes.html">attributes</a>.
     </p>
     <p>
-      Note: if the attribute field is an array of map or struct,
-      values are written in the summary store and update rates hence lower -
-      refer to <a href="https://github.com/vespa-engine/vespa/issues/10892">#10892</a> updates of array of map/struct.
+      Updating <a href="schemas.html#multivalue-fields">multivalue</a> attributes (arrays, maps, sets, tensors)
+      means reading the current value, making the update and writing it back:
+      <ul>
+        <li><a href="reference/schema-reference.html#type:weightedset">Weightedsets</a> and
```
Comment (geirst) [inline en/partial-updates.html:169]: Should probably mention arrays of primitive types as fast as well. That should also link to "reference/schema-reference.html#type:array".

File: en/partial-updates.html [hunk 2]
```diff
@@ -149,9 +163,17 @@ <h2 id="write-pipeline">Write pipeline</h2>
...
+        <a href="reference/schema-reference.html#type:tensor">tensors</a> are in memory and therefore fast,
+        see <a href="attributes.html#data-structures">attribute data structures</a> for performance considerations.</li>
+        <li>If the attribute field is an <a href="reference/schema-reference.html#type:array">array of map or struct</a>,
```
Comment (geirst) [inline en/partial-updates.html:172]: Rename to "array of struct" (with current link) or "map" (with another link: reference/schema-reference.html#type:map).

File: en/partial-updates.html [hunk 3]
```diff
@@ -149,9 +163,17 @@ <h2 id="write-pipeline">Write pipeline</h2>
...
+        values are written in the summary store and update rates are hence lower -
```
Comment (geirst) [inline en/partial-updates.html:173]: Rename "summary store" to "document store".

**Review by geirst — APPROVED** [review]
👍  Some comments to look at.

**Comment by kkraune** [comment]
thanks, added feedback!

**Review by geirst — COMMENTED** [review]
👍

**Pull request merged**